src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -98,7 +98,6 @@ export default class ConsentContinuationOut extends Mixins(CountriesProvincesMix
     const foreignJusrisdictionCountry = this.filing.data?.consentContinuationOut?.country
     const country = this.getCountryName(this.filing.data?.consentContinuationOut?.country)
     const region = this.getRegionNameFromCode(this.filing.data?.consentContinuationOut?.region)
-
```

Comment:
This blank line was intentional, to separate the variable declarations from the function code. Please revert it.

Reply:
TY